kolibri_explore_plugin/assets/src/components/DiscoveryNavBar.vue

```diff
@@ -58,6 +60,9 @@
       logo() {
         return assets.EndlessLogo;
       },
+      showDiscoveryTab() {
+        return !plugin_data.hideDiscoveryTab;
```

Comment:
If internally the semantics of this parameter is positive, perhaps it's better to switch the parameter itself to a positive semantic as well? I.e. `plugin_data.showDiscoveryTab` and the preference in the previous commit.

Reply:
Good point, I started switching the parameter and then realized that the internal semantic of the parameter in the last commit is negative. So the redirects would have to be `redirect: !plugin_data.showDiscoveryTab ? ...` ie "redirect if not show discovery tab".

Here is the context of this requirement: https://phabricator.endlessm.com/T34748#974595 . According to it, I think is better to have a "please hide the tab" setting as an exception for the online instance. Rather than a "show the tab" that's generally what we want.